Sources/Transitions/TransitionExtentions.swift

```diff
@@ -56,20 +56,20 @@ internal extension UIView {
 internal extension UILabel {
```

Comment:
Is `UITextView` extension missing to allow `fadeIn` / `fadeOut` like `UILabel`?

Reply:
This is fixed in latest commit.

Reply:
Even though it worked when I tested it (I think UITextView faded correctly from the UIView fade implementation, but it is better if it fades the textColor like the UILabel instead of fading opacity on sublayers.)